charts/vald/templates/agent/statefulset.yaml

```diff
@@ -93,6 +93,14 @@ spec:
           volumeMounts:
             - name: {{ $agent.sidecar.name }}-config
               mountPath: /etc/server/
+            {{- if not $agent.ngt.enable_in_memory_mode }}
+            {{- if $agent.ngt.index_path }}
+            {{- if $agent.persistentVolume.enabled }}
+            - name: {{ $agent.name }}-pvc
+              mountPath: {{ dir $agent.ngt.index_path }}
+            {{- end }}
```

Comment:
I think it would be better to automatically mount the emptyDir using if-else if persistentVolume is disabled, to avoid the problem of mounting the same volume-name mounting over each other.

Reply:
I don't think so.
The volume type is not only persistentVolume and emptyDir. If I'd like to mount a volume of hostPath type or something as an index storage, how do you deal with it?

Reply:
I thought it would not be a problem because writing to hostPath is deprecated for security reasons, and for other VolumeType, CSI is provided in many cases, and I think it can be used as PV.

Reply:
Okay. :+1:
Could you make an another PR about it? It is not a scope of this PR.

Reply:
Okay.
However, my idea may conflict with users who have already set up their own emptyDir in volumeMounts list, so this idea seems to have low priority.

I'll approve this PR.

Reply:
I created icebox ticket in our project, but it's low priority.
https://github.com/vdaas/vald/projects/1#card-65786136